components/date-picker/internal/DatePickerTemplate.razor

```diff
@@ -3,7 +3,7 @@
 @inherits DatePickerPanelBase<TValue>
 @using AntDesign
 
-<div class="@(PrefixCls)-@(PickerTypeMap[Picker])-panel" @ref="Ref">
+<div class="@(PrefixCls)-@PickerTypeMap[Picker]-panel @(ShowWeek ? "show-week-panel" : string.Empty)" @ref="Ref">
```

Comment:
Please follow the class name of antd
<img width="1271" alt="image" src="https://github.com/user-attachments/assets/9807ef3a-f5e4-40bd-89fe-66161fb99906" />